diagnostic.md

```diff
@@ -6,77 +6,92 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+A backend provides acccess to information in a database for a client. It is responsible for interpreting requests, finding and possibly changing the
+appropriate data, and then returning the results to the user.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+The model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+The controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+Views are not necessary serverside in an SPA. The client handles the view.
 ```
 
 What does C.R.U.D stand for?
 
 ```bash
-// your response here
+Create, Read, Update, Delete
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```bash
-// your response here
+The actions take place in the model, when it accesses the appropriate database.
```

Comment:
Actions are in the controller. From there it tells the model what to do to the DB